packages/management/addons-manager-v2/bundle/config/values.yaml

```diff
@@ -8,6 +8,7 @@ tanzuAddonsManager:
   deployment:
     hostNetwork: true
     priorityClassName: null
+    nodeSelector: null
     tolerations: []
```

Comment:
i think it is okay if we don't specify these attributes

Reply:
It is required by https://github.com/vmware-tanzu/tanzu-framework/issues/1947 to set default nodeSelector and tolerations for supervisor cluster.